### PR TITLE
Aadhaar button error + Learner date of birth age validation + Learner Aspiration make multiple check box

### PR DESCRIPTION
**Pull request by sagarkoshti1990**
- Aadhaar button error solved.
- Learner date of birth validation 12 to 30 years.
- Learner Aspiration make multiple check box for Motivation to pass 10th, career aspiration and support from Pragati camp

### CODE REVIEW

File: apps/front-end/src/pages/front-end/ag-form/Agform.js [hunk 1]
```diff
@@ -404,6 +317,20 @@ export default function Agform({ userTokenInfo, footerLinks }) {
         setotpbtn(false);
       }
     }
+    if (id === "root_dob") {
+      if (data?.dob) {
+        const today = moment().diff(data?.dob);
+        if (today < 378762675632 || today >= 946842558914) {
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/ag-form/Agform.js:323]: why is this hardcoded?

File: apps/front-end/src/pages/front-end/ag-form/Agform.js [hunk 2]
```diff
@@ -230,36 +182,6 @@ export default function Agform({ userTokenInfo, footerLinks }) {
     }
   };
 
-  const getOptions = (schema, { key, arr, title, value, filters } = {}) => {
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/ag-form/Agform.js:233]: this is not needed? not used anywhere?

File: apps/front-end/src/pages/front-end/ag-form/Agform.js [hunk 3]
```diff
@@ -122,22 +85,13 @@ export default function Agform({ userTokenInfo, footerLinks }) {
...
       } else if (pageStape.toLowerCase() === "n") {
-        await formSubmitUpdate({ ...formData, form_step_number: "13" });
         setPage("upload");
       } else {
         return true;
       }
     }
   };
 
-  const getLocation = () => {
```
Comment (manojLondhe) [inline apps/front-end/src/pages/front-end/ag-form/Agform.js:133]: this is not needed? not used anywhere?

File: apps/front-end/src/pages/front-end/ag-form/Agform.js (hunk 1 above)
Comment (sagarkoshti1990) [reply apps/front-end/src/pages/front-end/ag-form/Agform.js:323]: This is time difference 12 years and 30 years.

File: apps/front-end/src/pages/front-end/ag-form/Agform.js (hunk 2 above)
Comment (sagarkoshti1990) [reply apps/front-end/src/pages/front-end/ag-form/Agform.js:233]: This use but already move to `helper.js`

File: apps/front-end/src/pages/front-end/ag-form/Agform.js (hunk 1 above)
Comment (manojLondhe) [reply apps/front-end/src/pages/front-end/ag-form/Agform.js:323]: lets not hardcode epoch time, do something like this

```
let dob_date = moment([1994, 1, 1]);
let now_date = moment();
let age_in_years = now_date.diff(dob_date, 'years', true);

if (age_in_years >= 12 && age_in_years <= 30 ) {
    alert('age is in 12-30 range: ' + age_in_years);
}
else {
        alert('age is not in 12-30 range: ' + age_in_years);
}
```

**Pull request merged**